packages/web-react/src/components/Tabs/TabItem.tsx

```diff
@@ -1,16 +1,26 @@
 import React from 'react';
-import { ChildrenProps, TabId, TransferProps } from '../../types';
+import { ChildrenProps, TabId, TransferProps, ClickEvents, ClickEvent } from '../../types';
 import { useTabContext } from './TabContext';
 import { useTabsStyleProps } from './useTabsStyleProps';
 
-interface TabItemProps extends ChildrenProps, TransferProps {
+interface TabItemProps extends ChildrenProps, TransferProps, ClickEvents {
   forTab: TabId;
 }
 
-const TabItem = ({ children, forTab, ...restProps }: TabItemProps): JSX.Element => {
-  const { selectTab, selectedTabId } = useTabContext();
+const TabItem = ({ children, forTab, onClick, ...restProps }: TabItemProps): JSX.Element => {
+  const { selectTab, selectedTabId, onSelectionChange } = useTabContext();
   const { classProps } = useTabsStyleProps({ forTab, selectedTabId });
 
+  const clickHandler = (event: ClickEvent) => {
+    selectTab(forTab);
+    if (onClick) {
+      onClick(event);
+    }
+    if (onSelectionChange) {
```

Comment:
A new line before both conditions for better readability, please.

Reply:
fixed